hil/cli.py

```diff
@@ -294,6 +294,14 @@ def serve_networks():
         sleep(sleep_time)
 
 
+@cmd
+def list_create():
+    """List all users"""
```

Comment:
why is it list_create? `list_users` would me more appropriate.

Reply:
Also, the docstring could be more comprehensive. """List all users when database authentication is active""" or something along those lines.

Reply:
🤦‍♂️ I have no idea why it is `list_create`. Changing it now.

sure, I will put more verbose docstring.